businesscentral/image.yaml

```
@@ -106,6 +106,12 @@ envs:
     - name: "WORKBENCH_ROUTE_NAME"
       example: "myapp-rhpamcentr"
       description: "Route name that will be used to query its public address in the Kubernetes API to set the org.appformer.m2repo.url property. This env will only take effect if WORKBENCH_ROUTE_NAME is not empty. To set it to use https use the secure route available in the application template. Usually its name is secure-${APPLICATION_NAME}-rhpamcentr"
+    - name: "KIE_M2_REPO_DIR"
+      example: "/my/persistent/m2/volume"
+      description: "If KIE_PERSIST_MAVEN_REPO is set to true, it will be use to store Business Central maven artifacts. Default value is /opt/kie/data/m2"
```

Update the default location of .m2 dir (also mentioned in jira description) 
```suggestion
      description: "If KIE_PERSIST_MAVEN_REPO is set to true, it will be use to store Business Central maven artifacts. Default value is /home/jboss/.m2"
```

---

Or this default value is meant to be set to /opt/kie/data/m2 when is set KIE_PERSIST_MAVEN_REPO to true?

---

the default is only when it is enabled.